src/timer.h

```diff
@@ -75,7 +75,7 @@ extern const char *elapsed_str(DWORD val);
   extern void delay (unsigned int msec);
 #endif
 
-#if (DOSX) && defined(HAVE_UINT64)
+#if defined(WIN32) || defined(HAVE_UINT64)
```

Comment:
I like a `_WIN32` more  (since that is a built-in in MSVC, MinGW etc.). Doesn't OpenWatcom have that?